docs/topics/con_purpose-of-the-relational-database-backend-pattern.adoc

```diff
@@ -0,0 +1,32 @@
+[id=purpose-of-the-relational-database-backend-pattern_{context}]
+= Purpose of the relational database backend pattern
+
+This example demonstrates the ability of the HTTP application to locate and connect to a database in OpenShift.
+Each runtime shows how to implement the connectivity solution best suited in the given case.
+The runtime can choose between options such as using _JDBC_, _JPA_, or accessing _ORM_ APIs directly.
+
+The example application exposes an HTTP API, which provides endpoints that allow you to manipulate data by performing  _CRUD_ operations over HTTP.
+The _CRUD_ operations are mapped to HTTP `Verbs`.
+The API uses JSON formatting to receive requests and return responses to the user.
+The user can also use a user interface provided by the example to use the application.
+Specifically, this example provides an application that allows you to:
+
+* Navigate to the application web interface in your browser.
+This exposes a simple website allowing you to perform _CRUD_ operations on the data in the `my_data` database.
+* Execute an HTTP `GET` request on the `api/fruits` endpoint.
+* Receive a response formatted as a JSON array containing the list of all fruits in the database.
+* Execute an HTTP `GET` request on the `api/fruits/*` endpoint while passing in a valid item ID as an argument.
+* Receive a response in JSON format containing the name of the fruit with the given ID.
+If no item matches the specified ID, the call results in an HTTP error 404.
+* Execute an HTTP `POST` request on the `api/fruits` endpoint passing in a valid `name` value to create a new entry in the database.
+* Execute an HTTP `PUT` request on the `api/fruits/*` endpoint passing in a valid ID and a name as an argument.
+This updates the name of the item with the given ID to match the name specified in your request.
+* Execute an HTTP `DELETE` request on the `api/fruits/*` endpoint, passing in a valid ID as an argument.
+This removes the item with the specified ID from the database and returns an HTTP code `204` (No Content) as a response.
+If you pass in an invalid ID, the call results in an HTTP error `404`.
+
+ifndef::built-for-nodejs[]
+This example also contains a set of automated xref:running-the-example-application-integration-tests_{context}[integration tests] that can be used to verify that the application is fully integrated with the database.
+endif::built-for-nodejs[]
+
+This example does not showcase a fully matured RESTful model (level 3), but it does use compatible HTTP verbs and status, following the recommended HTTP API practices.
```

Comment:
This sentence is present in a section [3.3.1. Purpose of the relational database backend pattern](https://deploy-preview-1332--launcher-documentation.netlify.com/docs/vertx-runtime.html#purpose-of-the-relational-database-backend-pattern_crud) twice. Please could you remove it from here or form docs/topics/assembly_crud-example-application.adoc.